proto/podium/api/v1/podium.proto

```diff
@@ -0,0 +1,371 @@
+  rpc GetAroundMember(GetAroundMemberRequest) returns (MemberListResponse) {
+    option (google.api.http) = {
+      get: "/l/{leaderboard_id}/members/{member_public_id}/around"
+    };
+  }
+
+  // GetAroundScore retrieves the closest members to a score on the leaderboard.
+  rpc GetAroundScore(GetAroundScoreRequest) returns (MemberListResponse) {
+    option (google.api.http) = {
+      get: "/l/{leaderboard_id}/scores/{score}/around"
+    };
+  }
+
+  // GetTopMembers retrieves the top ranking members of a leaderboard.
+  rpc GetTopMembers(GetTopMembersRequest) returns (MemberListResponse) {
+    option (google.api.http) = {
+      get: "/l/{leaderboard_id}/top/{page_number}"
+    };
+  }
+
+  // GetTopPercentage retrieves a percentage of the top members of the leaderboard.
+  rpc GetTopPercentage(GetTopPercentageRequest) returns (MemberListResponse) {
+    option (google.api.http) = {
+      get: "/l/{leaderboard_id}/top-percent/{percentage}"
+    };
+  }
+
+  // UpsertScoreMultiLeaderboards sends a member score to multiple leaderboards.
+  rpc UpsertScoreMultiLeaderboards(MultiUpsertScoreRequest) returns (MultiUpsertScoreResponse) {
+    option (google.api.http) = {
+      put: "/m/{member_public_id}/scores"
+      body: "score_multi_change"
+    };
+  }
+
+  // GetRankMultiLeaderboards retrieves information about a member in multiple leaderboards.
+  rpc GetRankMultiLeaderboards(MultiGetRankRequest) returns (MultiGetRankResponse) {
+    option (google.api.http) = {
+      get: "/m/{member_public_id}/scores"
+    };
+  }
+}
+
+message HealthCheckResponse {
+  string working_string = 1;
+}
+
+message StatusResponse {
+  double error_rate = 1;
+}
+
+message RemoveLeaderboardRequest {
+  string leaderboard_id = 1;
+}
+
+// ScoreUpsert allow to provide score information about a single member.
+message ScoreUpsert {
+  string publicID = 1;
+  int64 score = 2;
+}
+
+// ScoreUpserts represent multiple score submissions.
+message ScoreUpserts {
+  repeated ScoreUpsert members = 1;
+}
+
+message BulkUpsertScoresRequest {
+  string leaderboard_id = 1;
+  bool prev_rank = 2;
+  int32 scoreTTL = 3;
+  ScoreUpserts score_upserts = 4;
+}
+
+// MemberResponse represents the full leaderboard information about a member.
+message MemberResponse {
+  string publicID = 1;
+
+  //implemented as double because int64 is exported as string through grpc-gateway
+  double score = 2;
+
+  //the full int64 score received from Redis
+  int64 int_score = 3;
+
+  int32 rank = 4;
+  int32 previous_rank = 5;
+  int32 expire_at = 6;
+  int32 position = 7;
+}
+
+// ScoreChange is the score payload when upserting a score.
+message ScoreChange {
+  int64 score = 1;
+}
+
+message UpsertScoreRequest {
+  string leaderboard_id = 1;
+  string member_public_id = 2;
+  bool prev_rank = 3;
+  int32 scoreTTL = 4;
+  ScoreChange score_change = 5;
+}
+
+message TotalMembersRequest {
+  string leaderboard_id = 1;
+}
+
+message TotalMembersResponse {
+  bool success = 1;
```

Comment:
FYI: I assume this is here for legacy reasons, but generally we should rely on the response status instead.

Reply:
Yes =(

Reply:
I agree. It is here for legacy reasons. :/